airflow/runtimes/container_runtime.go

```diff
@@ -32,6 +32,9 @@ const (
 // the container runtime lifecycle.
 type ContainerRuntime interface {
 	Initialize() error
+	Configure() error
+	ConfigureOrKill() error
+	Kill() error
```

Comment:
Adding 3 new lifecycle methods here. The airflow hooks call these methods depending on the situation.